protocol/0028-GOVE-governance.md

```diff
@@ -131,6 +131,27 @@ If a proposal is created and later a different outcome is preferred by network p
 
 Which of these makes most sense will depend on the type of change, the timing of the events, and how the rest of the community votes for the initial proposal.
 
+## Cancelling a proposal
+
+A proposal can be cancelled under certain conditions by, and only by, the party which created the proposal. To avoid situations where the community may have acted upon proposals they "expected" to pass, a party can only cancel their proposal before a certain "cancellation threshold" is reached.
+
+This cancellation threshold is a fixed point before the proposal closing time. The duration of the period in which a proposal cannot be cancelled is defined by the network parameter `governance.proposal.cancellationThreshold` (a duration string defaulting to `24h0m0s`).
```

Comment:
Given that there are proposal types with 24h total time possible, I would suggest making the default for this at least a bit lower—something like 20h or 23h—so that a proposal with even only 1 day of voting at least has some window in which mistakes can be corrected.

Reply:
`23h`.